mne/source_space.py

```diff
@@ -1182,6 +1182,54 @@ def vertex_to_mni(vertices, hemis, subject, subjects_dir=None, mode=None,
     return apply_trans(xfm['trans'], data)
 
 
+##############################################################################
+# Volume to MNI conversion
+
+
+@verbose
+def aseg_vertex_to_mni(vertices, subject, mri_head_t, subjects_dir=None,
+                       verbose=None):
+    """Convert the array of vertices for a hemisphere to MNI coordinates.
+
+    Parameters
+    ----------
+    vertices : n_vertices x 3 array of float
+        The  coordinates (in m) of the vertices in head coo system
+    subject : string
+        Name of the subject to load surfaces from.
+    mri_head_t: Transform
+        mri head tranformation
+    subjects_dir : string, or None
+        Path to SUBJECTS_DIR if it is not set in the environment.
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see :func:`mne.verbose`
+        and :ref:`Logging documentation <tut_logging>` for more).
+
+    Returns
+    -------
+    coordinates : n_vertices x 3 array of float
+        The MNI coordinates (in mm) of the vertices
+
+    Notes
+    -----
+    This function requires either nibabel (in Python) or Freesurfer
+    (with utility "mri_info") to be correctly installed.
+    """
+    if not has_freesurfer() and not has_nibabel():
+        raise RuntimeError('NiBabel (Python) or Freesurfer (Unix) must be '
+                           'correctly installed and accessible from Python')
```

Comment:
Let's let `_read_talxfm` handle throwing errors. If what is there is not already sufficient, let's improve it there.

Reply:
ok